__tests__/cache.test.ts

```diff
@@ -1,4 +1,3 @@
-import 'jest-localstorage-mock';
```

Comment:
This has been moved to `jest.config.js` so that it's included everywhere, as we're using it in a few places now.